marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx

```diff
@@ -609,19 +609,13 @@ describe("Matching", () => {
 
     cy.log("**Don't allow values to ignore with special characters**");
     rulesetSingleModal.addValuesToListToIgnore("Word$1");
-    cy.findByText("Names must start with a letter and can contain letters, numbers, hyphens, and underscores.");
+    cy.findByText("Names must start with a letter or number and can contain letters, numbers, hyphens, and underscores.");
```

Comment:
Could we clean up/delete tests that are already covered in the RTL?

Reply:
@aniamy  Yeah I can do that, but if that work is done and working, is that a good idea ?